src/challenge/Challenge.tsx

```diff
@@ -469,7 +520,7 @@ class Challenge extends React.Component<ChallengeProps, ChallengeState> {
             {this.renderHeader()}
             <Allotment className="h-100" defaultSizes={[65, 35]}>
               <Allotment.Pane>
-                <Allotment vertical defaultSizes={[65, 35]}>
+                <Allotment vertical defaultSizes={this.props.typ === "canvas" || this.state.typInferred === ChallengeTypes.TYP_CANVAS? [50, 50] : [65, 35]}>
```

Comment:
not sure if this will achieve what you are hoping for. Are the default sizes applied even when you change challenges? I don't think the Challenge component is reloaded.

Reply:
Agreed, it doesn't ... is there any way of making the allotments resize to give more space if a canvas pane is showing then? I suppose we don't want to override the user's dragged choice is the problem; perhaps best leave as is for now.

Reply:
found a fix for this (adjusting min size seems to work)